docker/Dockerfile

```diff
@@ -90,7 +90,7 @@ RUN wget --quiet \
 ARG CONDA_CUDF_VER=22.04
 RUN conda install -c rapidsai -c conda-forge cudf=$CONDA_CUDF_VER python=3.8 -y
 
-ARG RAFT_VER=23.02
+ARG RAFT_VER=22.12
```

Comment:
Can we add some comments to mention we fixed the version?

Reply:
Sure, updated.